api_tests/tests/sanity.ts

```diff
@@ -6,6 +6,60 @@ import SwapFactory from "../src/actors/swap_factory";
 // ******************************************** //
 
 describe("Sanity", () => {
+    it(
+        "invalid-path-yields-405",
+        oneActorTest(async ({ alice }) => {
+            // @ts-ignore (access to private field 'client')
+            const promise = alice.cnd.client.post("swaps/hbit/hbit", {});
+            await expect(promise).rejects.toHaveProperty("name", "Error");
```

Comment:
These assertions are pretty bad. They will pass for whatever reason the code threw an error :/

It would be good to assert the actual full problem response that we get from `cnd` so we can assert the status and title.

Reply:
I don't think we control this do we. FTR this is what we get back
```
    +   "code": "ECONNREFUSED",
    +   "columnNumber": undefined,
    +   "config": Object {
    +     "adapter": [Function httpAdapter],
    +     "baseURL": "http://0.0.0.0:36697",
    +     "data": "{}",
    +     "headers": Object {
    +       "Accept": "application/json, text/plain, */*",
    +       "Content-Length": 2,
    +       "Content-Type": "application/json;charset=utf-8",
    +       "User-Agent": "axios/0.19.2",
    +     },
    +     "maxContentLength": -1,
    +     "method": "post",
    +     "timeout": 0,
    +     "transformRequest": Array [
    +       [Function transformRequest],
    +     ],
    +     "transformResponse": Array [
    +       [Function transformResponse],
    +     ],
    +     "url": "swaps/hbit/hbit",
    +     "validateStatus": [Function validateStatus],
    +     "xsrfCookieName": "XSRF-TOKEN",
    +     "xsrfHeaderName": "X-XSRF-TOKEN",
    +   },
    +   "description": undefined,
    +   "fileName": undefined,
    +   "lineNumber": undefined,
    +   "message": "",
        "name": "Error",
    +   "number": undefined,
    +   "stack": undefined,
```

Reply:
And if you are interested, warp returns 405 because, again, if I understand it correctly, warp maintains a hierarchy of errors and returns the highest one. I don't have the tab open where I read that now though.